site/templates/index.html

```diff
@@ -6,14 +6,29 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700;900&display=swap" rel="stylesheet">
+  
```

Comment:
Should we have these be local?

Reply:
tbh I think its a little more hassle than its worth, doing it this way gives us a bunch of things for free.